core/templates/pages/contributor-dashboard-page/modal-templates/question-suggestion-review-modal.component.ts

```diff
@@ -149,41 +153,43 @@ export class QuestionSuggestionReviewModalComponent
 
   edit(): void {
     this.ngbActiveModal.dismiss();
-    this.skillBackendApiService.fetchSkillAsync(
-      this.suggestion.change.skill_id).then((skillDict) => {
-      const modalRef = this.ngbModal.open(
-        QuestionSuggestionEditorModalComponent, {
-          size: 'lg',
-          backdrop: 'static',
-          keyboard: false,
-        });
+    const skillId = this.suggestion.change.skill_id;
+    if (skillId) {
```

Comment:
```suggestion
    if (!skillId) {
      return;
    }
```
Allows us to unnest the following code.

Reply:
Done